Solutions/Marain.UserNotifications.Client/Marain/UserNotifications/Client/Management/Resources/DeliveryChannel.cs

```diff
@@ -0,0 +1,22 @@
+﻿// <copyright file="DeliveryChannel.cs" company="Endjin Limited">
+// Copyright (c) Endjin Limited. All rights reserved.
+// </copyright>
+
+namespace Marain.UserNotifications.Client.Management.Resources
+{
+    /// <summary>
+    /// Values of the different Delivery Channels which are integrated.
+    /// </summary>
+    public enum DeliveryChannel
```

Comment:
OK...this seems to repeat the earlier class of the same name, but in a client-side project. This is slightly concerning, because it seems to bake a fixed list of delivery providers into the client-side API. What happens when we want to add more? I guess we can push out a new version of the library with additional members in this enum, and hope that nothing anywhere in the system is attempting to validate values of this type against a known list based on the older version. But I don't really like the centralization of this. I would prefer delivery channels to be identified by a URI or GUID. You could still provide discoverability by providing a class with properties of the same name, it would just avoid tying things to an enum, which feels a bit...fragile.

Reply:
Hi Ian, we have followed the previous pattern which follows the same class names that have the same properties for the Host and the Client-side project.

If we want to add more delivery channels (eg.Twillio), we add it to this list as we would need to add all relevant classes to support this delivery channel. So I dont think so we would be deprecating any of the current delivery channels anytime soon.

Reply:
But we don't need to hard code these into the client. As Ian said earlier, these are fundamentally just identified with strings, and having a static class with a set of const strings in the client for well-known channels is convenient, but requiring a new build of the SDK when you add a delivery channel is an unnecessary coupling.

Reply:
Okay, I understand the string vs enum approach much better now. 

Agreed, we would change this to rather use a string.

Reply:
Changes have been made to convert this into a class having delivery channels as constant values.